Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -59,9 +59,10 @@
                           standalonemuontracking*
                           trackingGlobalReco*
                           vertexreco)
-_globalreco_tracking_Phase1PU70 = globalreco_tracking.copy()
-_globalreco_tracking_Phase1PU70.replace(trackingGlobalReco, recopixelvertexing+trackingGlobalReco)
-eras.trackingPhase1PU70.toReplaceWith(globalreco_tracking, _globalreco_tracking_Phase1PU70)
+_globalreco_tracking_LowPU_Phase1PU70 = globalreco_tracking.copy()
```

Comment:
@slava77 No, I still intend to clean up the trackingPhase1PU70 after the trackingPhase1 is frozen for next year. Here (and in few other places) I just wanted to avoid copy-paste. I can split them or just rename them to `_LowPU` if you want (to me it's the same if I do it now or when cleaning up trackingPhase1PU70).